src/main/java/jeryl/fyp/model/util/SampleDataUtil.java

```diff
@@ -17,20 +18,23 @@
  * Contains utility methods for populating {@code FypManager} with sample data.
  */
 public class SampleDataUtil {
+
+    public static final ProjectStatus DEFAULT_PROJECT_STATUS = new ProjectStatus("YTS");
```

Comment:
It would be nice to have variable project statuses for the sample students, not just YTS.